src/js/decorations/space.js

```diff
@@ -1,19 +1,25 @@
 import textBetween from '../utils/text-between';
 import createDeco from '../utils/create-deco';
 
-export default (predicate = char => char === ' ') => {
+function defaultPredicate(char) {
```

Comment:
I think specifying a whole new function here is perhaps a bit of an onerous API for the consumer - maybe we could change this to a factory called `character` with a signature like:

```ts
(class: string) => (predicate: (candidate: string) => boolean) => Builder
```

This means `space` could be recreated using the new `character` builder (i.e. `const space = character('space')`) and then exported to keep the same API.

Reply:
Do you mean then that the user would need to create multiple builders to handle multiple single-space characters.

Reply:
No not necessarily as you I think the `character` builder should also be exported from the lib - it would function almost exactly like your PR - it just means that `space` could _also_ be exported as a an already-constructed `character` builder. Does that make sense?

Ultimately it seems less surprising if all of the builders work off the idea of predicates. If one builder uses a `a -> str` as it's input predicate, whereas the others use `a -> bool` (a normal predicate)  it seems like this could catch someone out.

Reply:
Oh, I see. I updated the PR. Is that what you meant?